### PR TITLE
Run linter on build

### PR DESCRIPTION
**Pull request by chrisns**

### CODE REVIEW

File: api/controllers/Depmu_EntryController.js [hunk 1]
```diff
@@ -39,7 +39,7 @@ module.exports = {
         _.merge(prebooking, result)),
 
   filterContingencyWithoutCid: prebooking =>
-  !prebooking.contingency || (prebooking.contingency && _.isNull(prebooking.cid_id)),
+  !prebooking.contingency || prebooking.contingency && _.isNull(prebooking.cid_id),
```
Comment (cksanders) [inline api/controllers/Depmu_EntryController.js]: revert this change, as this is suppose to read: 
!prebooking.contingency 
OR
(prebooking.contingency && _.isNull(prebooking.cid_id)),

Comment (chrisns) [inline api/controllers/Depmu_EntryController.js]: you should find that is the boolean logic i'm expressing, the added brackets weren't needed which is why it doesn't fail any unit tests

Comment (cksanders) [inline api/controllers/Depmu_EntryController.js]: more for readability. Hopefully we are able to review our eslint setting at a later point, potential to update a few rules, particularly http://eslint.org/docs/rules/no-extra-parens
